zip/zip/z_fileio.c

```diff
@@ -3876,7 +3876,7 @@ int insert_arg(pargs, arg, at_arg, free_args)
    argnum = 0;
    newargnum = 0;
    if (args) {
-     for (; args[argnum] && argnum < at_arg; argnum++) {
+     for (; argnum < at_arg && args[argnum]; argnum++) {
```

Comment:
I suppose this is not important fix because it is a 3rd party component.

Reply:
Indeed but i don’t think it’d trigger some pb to include this change.